Core/Translation/en_EN.json

```diff
@@ -672,5 +672,17 @@
     "full-path": "Full path",
     "uploaded-at" : "Uploaded at",
     "hash": "Hash",
-    "field-can-not-be-null": "Field %fieldName% on table %tableName% can't be null"
+    "field-can-not-be-null": "Field %fieldName% on table %tableName% can't be null",
+    "delete-data": "Delete data",
+    "order-code": "Order code",
+    "combination-code": "Combination code",
+    "lineal-discount": "Lineal discount",
+    "pvp-unit": "Unit retail price",
+    "pvp-without-discount": "Retail price without discount",
+    "pvp-total": "Total retail price",
+    "delivery-notes-lines": "Delivery notes lines",
```

Comment:
We have lot of translations, please, use more generic words like "lines", instead of delivery note lines.